cpp/ql/src/semmle/code/cpp/Specifier.qll

```diff
@@ -13,6 +13,8 @@ class Specifier extends Element, @specifier {
     result instanceof UnknownDefaultLocation
   }
 
+  override string getCanonicalQLClass() { result = "Specifier(" + this.getName() + ")" }
```

Comment:
There are still some attributes and specifiers that need to have this parenthesis removed. Search for `")"` in the diff to find the others.